src/pages/Homepage/Homepage.vue

```diff
@@ -24,7 +24,7 @@ const TopMessageContentful = () => import('./TopMessageContentful');
 
 const activePageQuery = gql`query homepageFrame {
 	contentful {
-		entries(contentType: "page", contentKey: "home")
+		entries(contentType: "page", contentKey: "home-default")
```

Comment:
I think we'll want to keep the page as "home", but this is fine for now to test out the default page layout. I can handle updating this as a part of GROW-432.